pkg/security/admission/scc_exec.go

```diff
@@ -53,7 +53,7 @@ func (d *sccExecRestrictions) Admit(a admission.Attributes) (err error) {
 	// we're allowed to use the SA the pod is using.  Otherwise, user-A creates pod and user-B (who can't use the SA) can exec into it.
 	createAttributes := admission.NewAttributesRecord(pod, nil, kapi.Kind("Pod").WithVersion(""), a.GetNamespace(), a.GetName(), a.GetResource(), "", admission.Create, a.GetUserInfo())
 	if err := d.constraintAdmission.Admit(createAttributes); err != nil {
-		return admission.NewForbidden(a, fmt.Errorf("%s operation is not allowed because the pod's security context exceeds your permissions: %v", a.GetSubresource(), err))
+		return admission.NewForbidden(createAttributes, fmt.Errorf("%s operation is not allowed because the pod's security context exceeds your permissions: %v", a.GetSubresource(), err))
```

Comment:
> This seems weird too since it won't be the right action to the user.

Why it's not the right action? He is trying to get a shell and getting a forbidden error now. It's better than an internal error that he had before.

> Did you chase the problem down into the existing NewForbidden method yet?

I remember that I was close and I thought that the current approach feets perfectly.